test_requirements.txt

```diff
@@ -8,5 +8,9 @@ pytest-xdist
 # needed by pytest-xdist
 pytest >= 4.4.0
 
+# needed for enabling SymEngine
+numpy
+symengine==0.5.0
+
```

Comment:
I'd recommend not adding this here, but instead either:

* Adding a `test_symengine_requirements.txt` and a call to `pip install -r` conditional on it being this job
* Add a direct call to `pip install numpy symengine==0.5.0` on the symengine job

Reply:
Done